### ISSUE TITLE
a11y: Not enough color contrast in Try it Now example data response

### ISSUE THREAD

**Issue by ghost** [swagger-api/swagger-ui#9586]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]  Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic/ default

### Describe the bug you're encountering
We are using a tool called Sort Site to evaluate our site for 508 accessibility compliance.  It is flagging there is not enough color contrast in some of the sample values for the example response to the Try it Out: "Ensure that text and background colors have enough contrast."
Violates WCAG 2.0 AA 1.4.3 https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
This is in the span element with "style=color: rgb(211,99,99)"

### To reproduce...
Open a POST method, click Try it Now and click Execute.  Scroll down to the example value in the response.

Note this is just integers, other datatypes may also have problems

### Expected behavior
enough color contrast.

### Screenshots
Here is where it is happening:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/f24e8127-fe08-46a4-9ff8-680d943fb15f)

Here is the Sort Site error:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/acdd67ba-22c5-4979-b92b-7ddd7fb12794)

### Additional context or thoughts
Why are these data values shown in red anyway?  Red is the color for warnings and errors and there is nothing wrong with these data values.  Really, all data values should be rendered in the same green or blue color.
